build.gradle

```diff
@@ -173,8 +175,8 @@ dependencies {
      */
     compile 'org.springframework.boot:spring-boot-starter-data-jpa'
 
-    compile 'org.hibernate:hibernate-entitymanager'
-    compile 'org.hibernate:hibernate-core'
+    compile('org.hibernate:hibernate-entitymanager:5.1.16.Final')
+    compile('org.hibernate:hibernate-core:5.1.16.Final')
```

Comment:
same remark as in the other project, this should be managed by parent bom

Reply:
done and parent-bom updated